entries/QUnit.config.xml

```diff
@@ -50,7 +55,7 @@
 		</property>
 		<property name="testId" type="Array" default="undefined">
 			<desc>
-				This property allows QUnit to run specific test blocks by a hashed string combining their module name and test name. You can specify one or multiple tests to run.
+				This property allows QUnit to run specific tests identified by the hashed version of their module name and test name. You can specify one or multiple, comma-separated tests to run.
```

Comment:
I prefer "You can specify one or multiple tests to run" over "You can specify one or multiple, comma-separated tests to run". The type is already documented as Array, so "comma-separated" adds confusion rather than clarity (same goes for `moduleId` documentation).